logging/cluster-logging-release-notes.adoc

```diff
@@ -1,17 +1,46 @@
 [id="cluster-logging-release-notes"]
-= Release notes for Red Hat OpenShift Logging 5.1
+= Release notes for Red Hat OpenShift Logging
 include::modules/common-attributes.adoc[]
 :context: cluster-logging-release-notes-v5x
 
 toc::[]
 
-[id="openshift-logging-about-this-release"]
-== About this release
+[id="openshift-logging-inclusive-language"]
+== Making open source more inclusive
```

Comment:
I believe that this section is automatically included in documentation sets published on the Customer Portal based on an internally-available AsciiDoc snippet (unfortunately, I can't remember where it's defined). Including it here could potentially lead to a duplication of the section when published downstream.

As always, feel free to disregard this comment if I'm remembering this incorrectly, of course! :)

Reply:
Thanks for asking, Kevin. I did a quick search and see this statement simply pasted in a variety of locations. In the enterprise-4.* branches, it also appears in the OpenShift release notes.
```
Results for "Making open source more inclusive" in "openshift-docs/*.adoc"

/home/rdlugyhe/openshift-docs/jaeger/rhbjaeger-release-notes.adoc:
  12:  == Making open source more inclusive


/home/rdlugyhe/openshift-docs/logging/cluster-logging-release-notes.adoc:
  14:  == Making open source more inclusive


/home/rdlugyhe/openshift-docs/cicd/gitops/gitops-release-notes.adoc:
  21:  == Making open source more inclusive


/home/rdlugyhe/openshift-docs/cicd/pipelines/op-release-notes.adoc:
  22:  == Making open source more inclusive


/home/rdlugyhe/openshift-docs/service_mesh/v1x/servicemesh-release-notes.adoc:
  8:  == Making open source more inclusive


/home/rdlugyhe/openshift-docs/service_mesh/v2x/servicemesh-release-notes.adoc:
  8:  == Making open source more inclusive

```

Reply:
Having the same text copied and pasted across multiple different documents sounds like a recipe for trouble to me (having dealt with exactly that problem while working across the documentation set for the Go, Rust, and LLVM compilers). It's a maintenance problem, essentially.

By the way, I don't think that problem _needs_ to be addressed **right now** in this pull request. I do think that another issue should be opened at some point to discuss how this information can be presented in a more flexible and maintainable way, however. (Personally, I'd advocate for the notice itself being a reusable module or "snippet" if those are in use for OpenShift docs.)

Just something to think about! :)

Reply:
Good suggestion! I've created [RHDEVDOCS-3320 Create a "Making open source more inclusive" module for reuse](https://issues.redhat.com/browse/RHDEVDOCS-3320)